sticker-graph/sticker_graph/write_helper.py

```diff
@@ -19,18 +48,27 @@ def create_graph(config, model, args):
     tfconfig = tf.ConfigProto(gpu_options=gpuopts)
 
     with tf.Graph().as_default(), tf.Session(config=tfconfig) as session:
-        with tf.variable_scope("model", reuse=None):
-            model(config=config, shapes=shapes)
+        logdir = tf.placeholder(shape=[], name="logdir", dtype=tf.string)
+        summary_writer = _create_file_writer_generic_type(logdir)
+
+        with summary_writer.as_default(), tf.contrib.summary.always_record_summaries():
+            with tf.variable_scope("model", reuse=None):
+                model(config=config, shapes=shapes)
 
-        tf.variables_initializer(tf.global_variables(), name='init')
+            tf.group(tf.contrib.summary.summary_writer_initializer_op(),
+                     name="summary_init")
+            tf.variables_initializer(tf.global_variables(), name='init')
```

Comment:
Same question as above, does this need to be in that scope?

Reply:
It doesn't need to. I kept it so that the `graph_write` op will capture the full graph, including the init, save and restore ops.